dist/koemei-search-results.js

```diff
@@ -499,7 +507,7 @@ return /******/ (function(modules) { // webpackBootstrap
 	    };
 
 	    _this.engine.search(query, sync, async);
-	    !syncCalled && sync([]);
```

Comment:
@moeHaydar I don't remember but if I'm not wrong, I noticed that the check is useless because the two functions are called every time, either way.

Could you check the PR out and verify this point?